src/core/include/iDynTree/Core/VectorFixSize.h

```diff
@@ -205,66 +200,51 @@ namespace iDynTree
         }
         else
         {
-            memcpy(this->m_data,in_data,sizeof(double)*VecSize);
+            Eigen::Map<const Eigen::Matrix<double, VecSize, 1>> map(in_data, in_size);
+            this->operator=(map);
```

Comment:
Nit: any reason for not doing `*this = map;`? It seems more readable particularly for those that are not that familiar with operator overloading.

Reply:
```suggestion
            *this = map;
```